sentry-core/src/main/java/io/sentry/core/protocol/Device.java

```diff
@@ -290,6 +293,30 @@ public void setScreenHeightPixels(Integer screenHeightPixels) {
     this.screenHeightPixels = screenHeightPixels;
   }
 
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String id) {
+    this.id = id;
+  }
+
+  public String getLanguage() {
+    return language;
+  }
+
+  public void setLanguage(String language) {
+    this.language = language;
+  }
+
+  public String getConnectionType() {
```

Comment:
Is connection_type on that list of protocol additions? I recall id and language there

Reply:
yes, it is